tests/integration/helpers/a11y-audit-test.js

```diff
@@ -1,66 +1,63 @@
-import Component from '@ember/component';
-import { moduleForComponent, test } from 'ember-qunit';
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
 import hbs from 'htmlbars-inline-precompile';
 import a11yAudit from 'ember-a11y-testing/test-support/audit';
 
-// We use a component integration test to verify the behavior of the a11y-audit
-// by rendering a component and then running the audit on it.
-moduleForComponent('component:axe-component', 'Integration | Helper | a11y-audit', {
-  integration: true,
+module('Integration | Helper | a11yAudit', function(hooks) {
+  setupRenderingTest(hooks);
 
-  beforeEach() {
-    this.register('component:axe-component', Component.extend());
-  }
-});
-
-test('a11yAudit runs successfully with jquery context', function(assert) {
-  this.render(hbs`{{#axe-component}}{{/axe-component}}`);
+  test('a11yAudit runs successfully with jquery context', async function(assert) {
+    await render(hbs`{{#axe-component}}{{/axe-component}}`);
 
-  return a11yAudit(this.$()).then(() => {
+    await a11yAudit(this.$());
```

Comment:
Will need to tweak this test a bit, when running without jQuery (some of the default test scenarios now run without jQuery) this whole test should be removed/guarded (so that it doesn't run when jQuery is missing).

Reply:
@rwjblue I added a guard- can you check and see if it's correct?

Reply:
oops. I accidentally put it in the wrong place. Fixed it!